http_test.go

```diff
@@ -39,27 +39,6 @@ func CopyFile(dst, src string, perm os.FileMode) (err error) {
 	return
 }
 
-func Test_urlToFilename(t *testing.T) {
```

Comment:
I moved these tests to filecache [here](https://github.com/Nitro/filecache/pull/11/files#diff-7b8dd1da377a40dbf151f27edf254040R287).